README.md

```diff
@@ -10,6 +10,11 @@ Store this script locally and run it again every now and then to update your map
 curl https://raw.githubusercontent.com/tibiamaps/tibia-maps-installer-macos/master/install-tibia-maps > ~/bin/install-tibia-maps; chmod +x ~/bin/install-tibia-maps
 ```
 
+If you get `No such file or directory`, try saving the script on a different folder:
+```sh
+curl https://raw.githubusercontent.com/tibiamaps/tibia-maps-installer-macos/master/install-tibia-maps > your/path/install-tibia-maps; cd your/path; chmod +x install-tibia-maps
+
+```
```

Comment:
I’m not convinced this helps anyone. It might be better to prepend the above command with `mkdir -p ~/bin` (creating the `~/bin` dir if it doesn’t already exist). But still, people who don’t know their way around the shell should probably use the manual installation method.

Reply:
The issue for me was: /bin exists, ~/bin doesn't. I simply switched the ~/bin to /bin as that's the correct bin path on my osx version. Maybe a line telling them to create ~/bin as you said, or using whatever bin path they have would help? Not sure